docs/docs/upgrade/index.md

```diff
@@ -21,16 +21,38 @@ Upgrading to a newer version of Marathon should be executed in the following ord
 1. Start the instance with the new version of Marathon.
 1. Stop the last node with the old version. Now the new version of Marathon will take over leadership and becomes active.
 1. Watch the log of this instance for a successful start. There should be no ERROR or FATAL statements in the logs.
+1. Install the new version of Marathon on all remaining nodes with the old version.
```

Comment:
in the line below, please change to `Start all other instances of Marathon to build a quorum.`